cachito/workers/utils.py

```diff
@@ -19,25 +24,42 @@ def extract_app_src(archive_path, parent_dir):
     return os.path.join(parent_dir, 'app')
 
 
-def get_request_bundle_dir(request_id):
+class RequestBundleDir(paths.RequestBundleDir):
     """
-    Get the path to the directory where the bundle is being created for the request.
-
-    :param int request_id: the request ID to get the directory for
-    :return: the path to the where the bundle is being created
-    :rtype: str
+    Represents a concrete request bundle directory used on worker whose root
+    directory defaults to ``cachito_bundles_dir`` in config.
```

Comment:
Could you add a note here that says that the directories are created when this is instantiated?

Reply:
Could you also copy the parameters documentation from the base class' docstring?